_posts/blog/2019-08-13-Ang-Li-Three-People-in-Turing.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Three amazing people in Turing"
+categories: blog
+date: 2019-08-13
+modified:
+share: true
+author: Ang Li
+---
+
+Some amazing people I met in Turing. :)
+
+## Bo Wang
+
+Bo Wang is a visiting researcher at Turing and work with Professor Terry Lyons based at Oxford. 
+I knew him because he is one of the organizers for Data science for mental health, an interest group aiming
+to lead to suitable clinical interventions for mental health by using multi-modal data. Bo is interested in 
+developing clinically useful application for mental health by his expert in deep learning, also he’s working 
+on some other projects. One project he’s working on is to analyze speech signals of individuals with mental 
+disorders and try to help early identify clinical high risk individuals and aid the diagnose processing in 
+psychiatry. He believe the key of such success could benefit more from novel methods integrated with prior 
+knowledge and correct data, and we could not only rely on the magic of AI. He’s very helpful and helped me 
+with establishing the azure server and applying for credits, which is crucial for my work in Turing. Bo may 
+come back to China one day and It would be exciting if I can collaborate with him on some project in the future. 
+
+## Alex Campbell
+Alex Campbell is an enrichment student at Turing and a PhD candidate at University of Cambridge. During Master 
```

Comment:
"the Turing" and "the University go Cambridge", I recommend changing "During Master period" to "During his Masters"